.clang-format

```
@@ -0,0 +1,14 @@
+BasedOnStyle: "Google"
+
+AlignAfterOpenBracket:        Align
+AlignArrayOfStructures:       Left
+AlignConsecutiveAssignments:  AcrossComments
```

My personal preference is less aggressive alignment, like this or delete the whole line, because it causes more whitespace noise when making local changes. It's a classic bikeshed issue.
```suggestion
AlignConsecutiveAssignments:  Consecutive
```